src/clusterfuzz/_internal/bot/tasks/task_types.py

```diff
@@ -146,7 +165,7 @@ def execute(self, task_argument, job_type, uworker_env):
     'fuzz': UTaskLocalExecutor,
     'impact': TrustedTask,
     'minimize': UTaskLocalExecutor,
-    'progression': UTaskLocalExecutor,
+    'progression': ProgressionUTaskLocalExecutor,
```

Comment:
Can even call this stuff V2 instead of Progression, since it will soon be expanded.